common/bond.js

```diff
@@ -23,35 +23,41 @@ const bondBaseUrl = () => config.bondBaseUrl;
 // At some point we expect to have a more sophisticated way to do this, but for now, we have to do it this way.
 function determineBondProvider(urlString) {
     const url = URL.parse(urlString);
+
     if (DATASTAGE_NAMESPACES.includes(url.hostname.toLowerCase())) {
         return BondProviders.FENCE;
-    } else if (url.host.endsWith('.humancellatlas.org')) {
-        return BondProviders.HCA;
-    } else if (hasJadeDataRepoHost(url)) {
-        return BondProviders.JADE_DATA_REPO;
-    } else {
-        return BondProviders.default;
     }
+
+    if (ANVIL_NAMESPACE === url.hostname.toLowerCase()) {
+        return BondProviders.ANVIL;
+    }
+
+    if (url.host.endsWith('.humancellatlas.org')) {
+        return;
```

Comment:
I don't understand how you can return a const in one place and nothing in another. Doesn't this make it hard for the caller to use this function?

Reply:
This isn't returning nothing. It's a compact way of [returning `undefined`](https://codeburst.io/javascript-what-is-the-return-statement-97d8b11a1a0c). Yay for multiple empty types in JavaScript.

I'm still learning ES6 and the latest coding styles. [Here's a four year old discussion on `null` vs. `undefined`.](https://stackoverflow.com/questions/37980559/is-it-better-to-return-undefined-or-null-from-a-javascript-function) I would love to know best practices, and hopefully an `eslint` rule to enforce them so I don't forget.